src/Service/PaymentHandler.php

```diff
@@ -268,13 +275,13 @@ public function cancelPayment(string $hostedCheckoutId, int $amount, array $item
         $newStatus = $status;
         $amounts = [];
         $log = [];
+        $isFinal = $amount == $customFields[Form::CUSTOM_FIELD_WORLDLINE_PAYMENT_TRANSACTION_CAPTURE_AMOUNT];
```

Comment:
isFinal should be a bool, but it is probably an int, so the naming is strange here